resources/js/features/achievements/components/CreateAchievementTicketMainRoot/CreateAchievementTicketForm/EmulatorVersionField.tsx

```diff
@@ -24,7 +24,7 @@ export const EmulatorVersionField: FC = () => {
       name="emulatorVersion"
       render={({ field }) => (
         <BaseFormItem className="flex w-full flex-col gap-1 sm:flex-row sm:items-center">
-          <BaseFormLabel className="text-menu-link sm:mt-[13px] sm:min-w-36 sm:self-start">
+          <BaseFormLabel className="w-36 text-menu-link sm:mt-[13px] sm:min-w-36 sm:self-start">
```

Comment:
This field doesn't appear to be required, and trying to submit a ticket without it generates an error:
```
{
  "userId": 43495,
  "exception": "[object] (TypeError(code: 0): App\\Platform\\Data\\StoreTriggerTicketData::__construct(): Argument #6 ($emulatorVersion) must be of type string, null given, called in /var/www/html/app/Platform/Data/StoreTriggerTicketData.php on line 34 at /var/www/html/app/Platform/Data/StoreTriggerTicketData.php:15)
[stacktrace]
#0 /var/www/html/app/Platform/Data/StoreTriggerTicketData.php(34): App\\Platform\\Data\\StoreTriggerTicketData->__construct()
#1 /var/www/html/app/Platform/Controllers/Api/TriggerTicketApiController.php(24): App\\Platform\\Data\\StoreTriggerTicketData::fromRequest()
#2 /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): App\\Platform\\Controllers\\Api\\TriggerTicketApiController->store()
```

Reply:
Good find. I made a typo in a DTO. Resolved in latest.